Projects/Application/Sources/Flow/MainFlow/FeedScreen/FeedViewController.swift

```diff
@@ -28,5 +30,21 @@ final class FeedViewController: BaseViewController<FeedViewReactor, FeedView> {
                 cell.reactor = reactor
             }
             .disposed(by: disposeBag)
+
+        currentView.tableView.rx.contentOffset
+            .filter { [weak self] offset in
+                guard let `self` = self else { return false }
+                guard self.currentView.tableView.frame.height > 0 else { return false }
+                return offset.y + self.currentView.tableView.frame.height
+                    >= self.currentView.tableView.contentSize.height - 100
```

Comment:
Where does that magic `100` come from? Could you move it to a named constant?

Reply:
Just some random number, with finished post cell we can think about what is the good number here.